Octokit.Tests.Integration/Clients/IssueCommentsClientTests.cs

```diff
@@ -6,48 +6,59 @@
 using Octokit;
 using Octokit.Tests.Integration;
 using Xunit;
+using Octokit.Tests.Integration.Helpers;
 
 public class IssueCommentsClientTests
 {
-    private readonly IIssueCommentsClient _issueCommentsClient;
-
-    public IssueCommentsClientTests()
+    public class TheCreateMethod : IDisposable
     {
-        var github = Helper.GetAuthenticatedClient();
-        _issueCommentsClient = github.Issue.Comment;
-    }
+        private readonly IGitHubClient _github;
+        private readonly RepositoryContext _context;
 
-    [IntegrationTest]
-    public async Task CanDeserializeIssueCommentWhenGettingAllForRepository()
-    {
-        var comments = await _issueCommentsClient.GetAllForRepository("alfhenrik-test", "repo-with-issue-comment-reactions");
```

Comment:
Somewhat related - we have other tests here which use this repository, but this repository doesn't seem to exist any more cc @alfhenrik